extensions/Khronos/KHR_materials_cmnConstant/README.md

```diff
@@ -0,0 +1,118 @@
+`roughnessFactor`, should be ignored.
+
+```
+{
+    "materials": [
+        {
+            "name": "MyUnlitMaterial",
+            "pbrMetallicRoughness": {
+                "baseColorFactor": [ 0.5, 0.8, 0.0, 1.0 ]
+            },
+            "extensions": {
+                "KHR_materials_cmnConstant": {}
+            }
+        }
+    ]
+}
+```
+
+### Definition
+
+The Constant material model describes a constantly shaded surface that is
+independent of lighting. The material is defined only by properties already
+present in the [glTF 2.0 material specification](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#material).
+No new properties are added by this extension — it is effectively a boolean
+flag indicating use of an unlit shading model. Additional properties on the
+extension object are allowed, but may lead to undefined behaviour in conforming
+viewers.
+
+Color is calculated as:
+
+```
+color = <emissiveTerm> + <baseColorTerm> * <occlusionTerm> * <aL>
+```
+
+Term definitions:
+
+* **emissiveTerm** – `emissiveFactor` * `emissiveTexture`
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* **occlusionTerm** – `occlusionTexture.strength` * `occlusionTexture`
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#additional-maps)
+* **baseColorTerm** – `baseColorFactor` * `baseColorTexture` * `vertexColor`
+    *  [*From core glTF material specification.*](https://github.com/KhronosGroup/glTF/blob/master/specification/2.0/README.md#metallic-roughness-material)
+* **aL** – Indirect light
+    * *Constant sum of all ambient light contributions from the scene.*
+
+### Example
+
+The following example defines an unlit material with a constant red color.
+
+```json
+"materials": [
+    {
+        "name": "red_constant_material",
+        "pbrMetallicRoughness": {
+            "baseColorFactor": [ 1.0, 0.0, 0.0, 1.0 ]
+        },
+        "extensions": {
+            "KHR_materials_cmnConstant": {}
+        }
+    }
+]
+```
+
+### Extension compatibility and fallback materials
+
+When possible, authoring tools should provide a fallback material definition
+and mark the `KHR_materials_cmnConstant` extension as optional. Models
+including the extension optionally will still render in all clients that
+support the core glTF 2.0 specification, falling back to a PBR metal/rough
+material. In clients that do not support the extension, direct lighting will
+be applied and performance may be degraded.
+
+Defining a fallback to the default PBR shading model is achieved by setting
+appropriate parameters on the default material.
+
+```json
+"materials": [
+    {
+        "name": "unlit_with_fallback",
+        "pbrMetallicRoughness": {
+            "baseColorFactor": [ 1.0, 1.0, 1.0, 1.0 ],
+            "baseColorTexture": { "index": 0 },
+            "roughnessFactor": 1.0
```

Comment:
`metallicFactor` defaults to `1.0` and having a fully rough and fully metal material is not typical. Might consider setting `metallicFactor` to `0.0`.

Reply:
✅